gcs-action/__tests__/gcs.test.ts

```diff
@@ -0,0 +1,60 @@
+import * as gcs from '../src/gcs';
```

Comment:
I've migrated/migrating all the testing infrastructure from jest to mocha/chai. They should be mostly compatible, but can you take a look at https://github.com/GoogleCloudPlatform/github-actions/pull/60 and try to match that layout, file structure, and testing pattern please?

Reply:
Done